examples/kaleidoscope.nim

```diff
@@ -19,61 +19,65 @@ proc deallocCStringArray*(a: cstringArray) =
     inc(i)
     dealloc(a)
 
-{.emit: """
```

Comment:
Is Nim still case-insensitive? If not, this change may break LLVM < 11 since function names will all be different.

Reply:
I made a separated file for new versions